### ISSUE TITLE
Weekly build for Ubuntu 20.04

### ISSUE THREAD

**Issue by pimlie** [eranif/codelite#2547]
Hey, was wondering whether you are planning to also release weekly builds for Ubuntu 20.04? Its mainstream for a couple of months now and although 14.0.0 is available there are some nice fixes in 14.0.1

Thanks as always! :)

**Comment by eranif** [comment]
Are you able to run the 18.04 build on 20.04?
Also, I just uploaded 14.0.2 build

**Comment by pimlie** [comment]
> Are you able to run the 18.04 build on 20.04?

Yes, actually I was. Not sure what I did wrong yesterday, but codelite requires libhunspell-1.6 while Ubuntu 20.04 ships with 1.7. Tried installing [1.6](https://packages.ubuntu.com/bionic/amd64/libhunspell-1.6-0/download) yesterday but somehow failed (it was complaining about 2 other dependencies), but when trying again today it just worked.

Anyway, thanks & sorry for being a typical user ;)

> Also, I just uploaded 14.0.2 build

Awesome, can confirm this works with 20.04! :bowing_man:
